packaging/maintainership/en.md

```diff
@@ -0,0 +1,48 @@
+
+The role of a maintainer is to update package(s) on a regular basis, while following Solus packaging and consistency standards.
+
+## Commitment
+
+Each new package which is going to land in the Solus repository must have one or more maintainers. These individuals commit to:
+ - Update the package reasonably regularly, unless there are valid reasons to hold it back
+ - Ensure the [packaging file](/articles/packaging/package.yml/en) adheres to the Solus [standards](/articles/packaging/packaging-practices/en)
+ - Ensure the application or library is consistent with the Operating System aesthetics, file system conventions and the Solus philosophy in general
+
+On the Solus side however, the community must not forget that maintainers are volunteers, which may or may not have a technical background. More experiencied users are to engage newcoming maintainers in a welcoming manner, e.g. by listing their errors and inviting them to fix them. (More on this in the [community guidelines](/articles/contributing/community-guidelines/en.md#development--issue-trackers))
+
+The Core Team has the right to enforce certain practices, even when in contrast with maintainers' vision. It also has the right to ultimately accept or reject a patch.
+
+## Stepping in
+
+To officially step in as the maintainer of a package, a `MAINTAINERS.md` file must be provided for accepted packages that are not yet included in the repository, or if the package predates the policy of requiring one. Instead, if a previously maintained package is marked by the Solus team as needing a new maintainer, the `MAINTAINERS.md` file must be updated.
+
+## Updating a maintained package
+
+The procedure varies depending on whether or not the individual is the maintainer.
+
+### Maintainer individuals
+
+Maintainers are free to [update a package](/articles/packaging/updating-an-existing-package/en) at their will, unless otherwise indicated by the Core Team.
+
+### Non-maintainer individuals
+
+If a package is actively maintained, modifications should not occur and the individual should exercise patience when it comes to the updating of software. In some cases, a maintainer may intentionally be holding back a package, or has simply not updated yet. If pertinent, the individual should file a package update request. Alternatively, the individual can reach the maintainers or the Core Team on IRC and ask permission to update the package. It is also possible to submit a patch and attach a message to it clarifying the intention of updating the package, although this is a special case reserved to e.g. security updates and the patch is likely to be rejected.
+
+## Template
+
+Presented here is the `MAINTAINERS.md` file. This file must be provided verbatim alongside the other patch contents, and filled in with the maintainers' personal information. An IRC contact is optional but recommended, but an e-mail address is mandatory. For accountability purposes, packagers must also fill in their real names.
```

Comment:
"For accountability purposes, packagers must also fill in their real names."

Let's clarify it further with the content below, just so everyone is on the same page that it is already required as part of the .solus/packager file:

```
Similar to the `.solus/packager` file used for packaging, the maintainers listed in `MAINTAINERS.md` must use their real first and last name(s) for accountability purposes.
```

Reply:
`.solus/packager` or `~/.config/whatever` ?

Reply:
Our Building a Package doc references .solus/packager so let's stick with that for now.